frontend/src/theme/components/index.ts

```diff
@@ -24,6 +25,7 @@ export const components = {
   PhoneNumberInput,
   Textarea,
   [CHECKBOX_THEME_KEY]: Checkbox,
+  [RADIO_THEME_KEY]: Radio,
```

Comment:
regretting the theme key now, since they are technically only used once haha.
but since this is now exported, should it also be used in the component itself?